homework/hackyourtemperature/server.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+import express from 'express';
+import bodyParser from 'body-parser';
+
+let Server=express();
+
+Server.use(express.json());
+Server.use(bodyParser.urlencoded({ extended: true }));
+Server.post('/weather',(request,response)=>{
+  const cityName=request.body.cityName;
+  response.write(`Your city is ${cityName}`);
+  response.end();
+});
+Server.use('/',(request,response,next)=>{
```

Comment:
```suggestion
Server.use('/',(request,response)=>{
```

If you aren't using something it is mostly better to not include it.